### ISSUE TITLE
VBoxManage: error: Unknown option: --paravirtprovider

### ISSUE THREAD

**Issue by efenderbosch** [cloudfoundry/bosh-lite#363]
I'm trying to use 9000.105.0.

README.md specifies VirtualBox 4.3.14r95030. When I this version is installed, I get the following error:

```
VBoxManage --version
WARNING: The vboxdrv kernel module is not loaded. Either there is no module
         available for the current kernel (3.19.0-32-generic) or it failed to
         load. Please recompile the kernel module and install it by

           sudo /etc/init.d/vboxdrv setup

         You will not be able to start VMs until this problem is fixed.
```
4.3.12 also gives this error. Running vboxdrv setup doesn't complete successfully.

I've tried all other 4.3 versions from 16 through 30. They don't give that error, but I do get this error when I run ```vagrant up --provider=virtualbox```:

```VBoxManage: error: Unknown option: --paravirtprovider```

Output of ```uname -a```:

```3.19.0-32-generic #37~14.04.1-Ubuntu SMP Thu Oct 22 09:41:40 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux```

Vagrant is version 1.6.3, as specified in README.md.

Any direction to help fix this would be appreciated.

**Comment by cppforlife** [comment]
See https://github.com/cloudfoundry/bosh-lite/blob/master/docs/troubleshooting.md#customization-command-failed

**Comment by efenderbosch** [comment]
```VBoxManage --version``` returns  ```5.0.4_LinuxMintr102546```

vagrant up --provider=virtualbox returns:

```
The provider 'virtualbox' that was requested to back the machine
'default' is reporting that it isn't usable on this system. The
reason is shown below:

Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1, 4.2, 4.3
```

**Comment by efenderbosch** [comment]
I went and installed the latest vagrant and now it works. Looks like README.md should be updated with the correct versions.

**Comment by cppforlife** [comment]
https://github.com/cloudfoundry/bosh-lite is updated to reference 5.x virtualbox.

**Comment by vhosakot** [comment]
I saw the same error with vagrant 1.8.7 on CentOS 7.3 with kernel version 3.10.0-514.10.2.el7.x86_64.

```
The vboxdrv kernel module is not loaded. Either there is no module
         available for the current kernel (3.10.0-514.10.2.el7.x86_64)
```

The following steps resolved for me:
1. `yum -y install kernel-headers kernel-devel`
2. Reboot.
3. `sudo /sbin/rcvboxdrv setup`
4. `VBoxManage --version`
